e2e-test/e2e_test_suite_test.go

```diff
@@ -0,0 +1,86 @@
+package e2e_test
```

Comment:
nit: `_` as a package name is not encouraged in golang, e2etest could be a better substitute.

FYI: https://blog.golang.org/package-names